www/js/views/services.js

```diff
@@ -9,33 +9,44 @@
  */
 
 var $ = require('jquery');
-
+var _ = require('underscore');
 var Backbone = require('backbone');
 var adminui = require('../adminui');
 var Services = require('../models/services');
+var Model = require('../models/model');
+var Collection = require('../models/collection');
 var CollectionView = require('./collection');
+var DEFAULT_TYPE = 'all';
 
-var Instance = require('../models/model').extend({
+var Instance = Model.extend({
     idAttribute: 'uuid',
     urlRoot: '/api/instances'
 });
 
-var Instances = require('../models/collection').extend({
+var Instances = Collection.extend({
     model: Instance,
     url: '/api/instances'
 });
 
-var Application = require('../models/model').extend({
+var Application = Model.extend({
     idAttribute: 'uuid',
     urlRoot: '/api/applications'
 });
 
-var Applications = require('../models/collection').extend({
+var Applications = Collection.extend({
     model: Application,
-    url: '/api/applications'
+    url: '/api/applications',
+    filterByType: function (type) {
+        var applications = null;
+        if (type !== DEFAULT_TYPE) {
+            applications = this.filter(function (application) {
+                return application.attributes.name.toLowerCase() === type;
```

Comment:
The literals comparison is syntactically incorrect here. Regardless, the 'type' field currently has either 'vm' or 'agent' as value and will not help to distinguish between sdc vs manta services. We'll need to add in some new code to first get all the application uuid for application name = sdc (and manta). Then use that uuid to filter for services with matching application_uuid.